src/features/bookOffer/components/Calendar/Calendar.tsx

```diff
@@ -76,6 +88,13 @@ interface Props {
 export const Calendar: React.FC<Props> = ({ stocks, userRemainingCredit }) => {
   const { bookingState, dispatch } = useBooking()
   const stocksDate = getStocksByDate(stocks)
+  const debouncedDispatch = useRef(debounce(dispatch, 1000)).current
```

Comment:
```suggestion
  const debouncedDispatch = useRef(debounce(dispatch, 200)).current
```

sinon on dirait que c'est un bug